lib/election.js

```diff
@@ -1032,9 +1031,8 @@ function _createManifest(ledgerNodeId, blockHeight, type, items, callback) {
 }
 
 function _createVote(ledgerNode, electionTopic, vote, voter, callback) {
-  // TODO: add logger.verbose() with the vote information here
-  // console.log(voter.id.split('/').pop() + ' votes for ' + electionTopic +
-  //   ', manifestHash=' + vote.manifestHash + ', round=' + vote.voteRound);
+  logger.verbose(voter.id.split('/').pop() + ' votes for ' + electionTopic +
```

Comment:
@mattcollier,

This `voter.id.split('/').pop()` stuff probably needs to change to just `voter.id` (shouldn't assume too much about how these look) ... and we should use the meta data field with the logger and move and/or duplicate some of these items into logged meta data properties. I don't remember how that works, can you take a stab?

Reply:
@dlongley the api is (message, meta) where meta must be an object.  What do you want the logger message to say?